libs/vue/src/schematics/application/schematic.ts

```diff
@@ -173,17 +173,29 @@ function addJest(options: NormalizedSchema): Rule {
       testEnvironment: 'jsdom',
       babelJest: false,
     }),
-    updateJsonInTree(`${options.projectRoot}/tsconfig.spec.json`, (json) => {
-      json.include = json.include.filter((pattern) => !/\.jsx?$/.test(pattern));
-      json.compilerOptions = {
-        ...json.compilerOptions,
-        jsx: 'preserve',
-        esModuleInterop: true,
-        allowSyntheticDefaultImports: true,
-      };
-      return json;
-    }),
+    !options.isVue3
```

Comment:
Those options were originally added because Vue didn't process the `extend` property inside the `tsconfig.spec.json`. With Vue3, the extend works so these options are no longer needed. As for the `include` property, that shouldn't be modified so I'll revert that.